LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -6,7 +6,18 @@
 //--------------------------------------------------------------------------------------- 
 
 class UIOutpostManagement extends UIScreen
-    dependson(XComGameState_LWOutpost);
+    config(LW_UI) dependson(XComGameState_LWOutpost);
+
+// KDM : Additional Variables Start
```

Comment:
Not really needed. The obvious question is "additional to what?" I don't see value in this (unlike the policy of the Community Highlander, which is used specifically to identify what has been changed from the base game code).